Hackerrank3.py

```diff
@@ -0,0 +1,18 @@
+# Read a given string, change the character at a given index
+# and then print the modified string.
+# replace the character at index i with character c.
+
+
+def mutate_string(string, position, character):
+    spis = list(s)

```

Comment:
string position character

Reply:
This arguments where specified in the blok below #..
I couldn't delete this block on hackerrank, so I wrote the solution based on the code and variables that were written in this block.
Or I don't understand what you want me to dwell on.

Reply:
Inside function it's good to use parameters that were passed to function. Using global variables - variables that are defined outside function is pretty bad idea